### PR TITLE
🚑️ Fix typo in kf target_api_plugin

### PR DESCRIPTION
**Pull request by chris-s-friedman**
workflow_tool and workflow_type are misspelled

See [D3B-723](https://d3b.atlassian.net/browse/D3B-723)

[D3B-723]: https://d3b.atlassian.net/browse/D3B-723?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by HuangXiaoyan0106 — APPROVED** [review]

**Review by znatty22 — CHANGES_REQUESTED** [review]
Looks good! Although I see the python 3.9 unit tests failing bc of a dependency issue. The reason is this: https://stackoverflow.com/questions/78650222/valueerror-numpy-dtype-size-changed-may-indicate-binary-incompatibility-expec

I think the solution is to pin numpy in the requirements.txt file: `numpy<2.0.0`

**Review by znatty22 — APPROVED** [review]
@chris-s-friedman This will be good to go after its rebased on master (after the other PR is merged)

**Pull request merged**
